charts/testkube-enterprise/local-values.yaml

```diff
@@ -261,6 +276,14 @@ testkube-logs-service:
     nats:
       # -- NATS URI
       uri: "nats://testkube-enterprise-nats:4222"
+    minio:
```

Comment:
Why was this needed? Was the installer tested again? cc @ypoplavs

Reply:
Logs service is storing data in MinIO